tsl/src/remote/connection.c

```diff
@@ -1227,7 +1231,8 @@ remote_connection_open_with_options_nothrow(const char *node_name, List *connect
 					/* Should wait for write on socket */
 					events |= WL_SOCKET_WRITEABLE;
 					break;
-				default:
```

Comment:
An example of PGconnectPoll usage in PG keeps `default` case and also have check on `pollstatus != PGRES_POLLING_OK`. Will it be good to keep the default case and handle it in the same way as `PGRES_POLLING_FAILED`?